powerinfer-py/powerinfer/solver.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# coding=utf-8
+import argparse
+from cvxopt.glpk import ilp
+import numpy as np
+from cvxopt import matrix
+import torch
+import pickle
+
+def solve_gpu_split(
```

Comment:
Solver for which weights need to be CPU and which on GPU. Need to fit the vram budget.